INN/umbrella-inndev
# Learn post content sans serif

## Changes

This pull request makes the following changes:

- Updated `.entry-content` `font-family` style attribute to also be used for `ol` and `ul` elements inside of `entry-content`.

Before:
![Screen Shot 2019-07-05 at 12 04 54 PM](https://user-images.githubusercontent.com/18353636/60734193-1ebbc180-9f1d-11e9-91e6-81806cb78d72.png)

After:
![Screen Shot 2019-07-05 at 12 04 49 PM](https://user-images.githubusercontent.com/18353636/60734194-1ebbc180-9f1d-11e9-8bde-7640f62dc594.png)

## Why



For #98 

## Testing/Questions

Features that this PR affects:

- Learn site guide pages

Steps to test this PR:

1. View a guide with multiple content text elements, like [here](https://learn-inn-org.inndevlearn.wpengine.com/guides/business-resources/tools-nonprofit-news/taxes/) and make sure the font is all `sans-serif`.
